.gitignore

```diff
@@ -10,6 +10,7 @@ lerna-debug.log*
 dump/
 db/
 mongos/
+dump.rdb
```

Comment:
why is this added?

Reply:
@ijemmao 
Said file must have been generated whilst setting up the project. Refactored it, Thanks!